config/canonical.json

```diff
@@ -9144,6 +9155,8 @@
     "tags": {
       "amenity": "fuel",
       "brand": "Ceypetco",
+      "brand:wikidata": "Q5065795",
+      "brand:wikipedia": "en:Ceylon Petroleum Corporation",
       "name": "Ceypetco"
```

Comment:
This name possibly should be capitalised?

Reply:
Which name isn't? It looks like every use of it is. On Wikipedia its Ceylon Petroleum Corporation. Or do you mean the brand name should be in all capitals? The actual name is in all caps, but I figured id just leave it how it was. I guess I could change it though.

Reply:
Yeah I meant `CEYPETCO` as that's how they seem to write it, however I don't know if we prefer to just go with existing majority tagging

Reply:
Id say go with existing majority tagging. Thats what seems to be other  similar cases at least. Although I could be wrong. Does everything else look currect to you outside of those two things?